docker-compose.yml

```diff
@@ -6,8 +6,8 @@ services:
     build:
       context: .
       args:
-        USER_ID: $USER_ID
-        GROUP_ID: $GROUP_ID
+        USER_ID: ${USER_ID:-1001}
+        GROUP_ID: ${GROUP_ID:-1001}
```

Comment:
yes! This was the final piece of the puzzle! I was looking for this! Thanks!
I was getting this issue https://github.com/docker/compose/issues/3608